### PR TITLE
CLI command to check repos using bundler-audit

### PR DESCRIPTION
**Pull request by boffbowsh**

### CODE REVIEW

File: lib/govuk_security_audit.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require "govuk_security_audit/version"
+
+module GovukSecurityAudit
+  # Your code goes here...
```
Comment (jamiecobbett) [inline lib/govuk_security_audit.rb]: This probably doesn't need to be here.

File: exe/govuk_security_audit [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
```
Comment (jamiecobbett) [inline exe/govuk_security_audit:1]: Why `exe/`? `bin/`is more conventional?

**Comment by jamiecobbett** [comment]
Would you mind adding a `CHANGELOG.md`?

File: exe/govuk_security_audit (hunk 2 above)
Comment (boffbowsh) [inline exe/govuk_security_audit:1]: It's to separate gem-supplied exe files from bundler-generated binstubs http://bundler.io/blog/2015/03/20/moving-bins-to-exe.html

**Comment by boffbowsh** [comment]
Changelog added

**Pull request merged**